.github/workflows/rust.yml

```diff
@@ -25,5 +25,7 @@ jobs:
 
       - name: Check fmt
         run: cargo fmt -- --check
-      - name: Build
+      - name: Build (nightly)
         run: ./ci.sh
+      - name: Build (stable)
+        run: ./ci-stable.sh
```

Comment:
could you make this run with latest Rust stable? Currently it'll be picking up the nightly version from `rust-toolchain.toml`.

Reply:
done